src/main/java/edu/phema/elm_to_omop/vocabulary/translate/map/ActCodeToVisitConceptMap.java

```diff
@@ -0,0 +1,45 @@
+package edu.phema.elm_to_omop.vocabulary.translate.map;
+
+import edu.phema.elm_to_omop.vocabulary.phema.PhemaCode;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Translates the ActCode vocabulary to the OMOP internal visit concept vocabulary
+ * <p>
+ * Source: https://www.hl7.org/fhir/STU3/v3/ActEncounterCode/vs.html
+ * Target: http://athena.ohdsi.org/search-terms/terms?vocabulary=Visit
+ */
+public class ActCodeToVisitConceptMap implements IVocabularyMap {
+    private static Map<String, String> map;
+
+    static {
+        map = new HashMap<>();
+
+        // TODO: Have someone validate these mappings
+        map.put("AMB", "OP");
+        map.put("EMER", "ER");
+        map.put("FLD", "LCTP");
```

Comment:
I wasn't able to find the LCTP code in our OMOP database or a visit-related concept in ATHENA (http://athena.ohdsi.org/search-terms/terms?page=1&pageSize=15&query=LCTP).

Reply:
I should have tried different letter orders - looks like it should be LTCP - http://athena.ohdsi.org/search-terms/terms/42898160.  I don't think that applies in this case though.  LTCP sounds more like long-term care.  FLD sounds more like an EMS response, so maybe Ambulance Visit? http://athena.ohdsi.org/search-terms/terms/581478

Reply:
Thanks for reviewing. I'll use one of the codes you suggest.